src/mmrelay/meshtastic_utils.py

```
@@ -1175,12 +1336,60 @@ def connect_meshtastic(
+                                        "This may indicate a stale BlueZ connection or Bluetooth adapter issue."
+                                    )
+                                    logger.warning(
+                                        "Try: 1) Restarting BlueZ: 'sudo systemctl restart bluetooth', "
+                                        "2) Manually disconnecting device: 'bluetoothctl disconnect %s', "
+                                        "3) Rebooting your machine",
+                                        ble_address,
+                                    )
+                                    meshtastic_iface = None
+                                    raise TimeoutError(
+                                        f"BLE connection attempt timed out for {ble_address}."
+                                    ) from None
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)